csci_tool/commands/mutate.py

```diff
@@ -60,9 +60,10 @@ def clone(student):
         # source_dir is where the mutator files exist in the meta repo
         source_dir = path.join(meta_dir, mutation_name)
         # make the changes
-        for student, repo_path in repos:
+        for student, repo in repos:
             l.info('Mutating repo: %s', student.unix_name)
             cwd = os.getcwd()
+            repo_path = repo.git.rev_parse("--show-toplevel")
```

Comment:
We don't actually need the path if we already have an instantiated repo

Reply:
yea the changes I made to mutate were just to get it to work...but I think you fixed all of these problems already